### PR TITLE
INTYGFV-16420: Removing code handling subscription feature toggling.

### PR DESCRIPTION
**Pull request by mikeyOne**

### CODE REVIEW

File: web/build.gradle [hunk 1]
```diff
@@ -32,7 +32,7 @@ gretty {
     serverConfigFile = "tomcat-gretty.xml"
 
     jvmArgs = ["-Dcatalina.base=${buildDirectory}/catalina.base",
-               "-Dspring.profiles.active=dev,caching-enabled,wc-security-test,ia-stub,certificate-service-active,hsa-integration-intyg-proxy-service,servicenow-integration",
+               "-Dspring.profiles.active=dev,caching-enabled,wc-security-test,ia-stub,certificate-service-active,hsa-integration-intyg-proxy-service,wc-kundportalen-stub,wc-all-stubs",
```
Comment (mww-sopra-steria) [inline web/build.gradle:35]: Is it correct that we should use these profiles? Or were the left after testing?

File: web/src/main/java/se/inera/intyg/webcert/web/service/facade/impl/certificatefunctions/GetUserResourceLinksImpl.java [hunk 2]
```diff
@@ -29,12 +29,9 @@
 import se.inera.intyg.webcert.web.web.controller.facade.dto.ResourceLinkTypeDTO;
 
 @Service
+@NoArgsConstructor
```
Comment (mww-sopra-steria) [inline web/src/main/java/se/inera/intyg/webcert/web/service/facade/impl/certificatefunctions/GetUserResourceLinksImpl.java:32]: Isn't default constructor "no args" and this is redundant?

File: web/src/main/webapp/app/webcert.scss [hunk 3]
```diff
@@ -1,9 +1,12 @@
 @CHARSET "UTF-8";
 
 @import "mixins/wc-variables";
-@import "mixins/bootstrap-variables-override"; // Bootstrap override variables must be loaded before bootstrap due to its !default flags
-@import '../bower_components/bootstrap-sass/assets/stylesheets/bootstrap.scss'; // Load bootstrap defaults
-@import "mixins/bootstrap-css-override"; // CSS Override only, no bootstrap variables
+@import "mixins/bootstrap-variables-override";
```
Comment (mww-sopra-steria) [inline web/src/main/webapp/app/webcert.scss:4]: Better move the comment so it sits above the row the comment is for (or return it to the same line).

**Review by mww-sopra-steria — APPROVED** [review]
Great job cleaning it up! Approved, but look at the comments before merging.

File: web/build.gradle (hunk 1 above)
Comment (mikeyOne) [reply web/build.gradle]: Profile set to servicenow-intgration

File: web/src/main/java/se/inera/intyg/webcert/web/service/facade/impl/certificatefunctions/GetUserResourceLinksImpl.java (hunk 2 above)
Comment (mikeyOne) [reply web/src/main/java/se/inera/intyg/webcert/web/service/facade/impl/certificatefunctions/GetUserResourceLinksImpl.java]: Removed annotation

File: web/src/main/webapp/app/webcert.scss (hunk 3 above)
Comment (mikeyOne) [reply web/src/main/webapp/app/webcert.scss]: Comments now above the lines they comment

**Pull request merged**